docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -0,0 +1,133 @@
++++
+date = "2020-09-26T13:14:52-04:00"
+title = "Installing the mongocxx driver"
+[menu.main]
+  identifier = "mongocxx3-installation"
+  parent = "mongocxx3"
+  weight = 8
++++
+
+## Prerequisites
+
+- Any standard Unix platform, or Windows 7 SP1+
+- A compiler that supports C++11 (gcc, clang, or Visual Studio)
+- [CMake](https://cmake.org) 3.2 or later
+- [boost](https://www.boost.org) headers (optional)
+
+We currently test the driver on the following platforms:
+
+- Linux with clang 3.8 and 6.0, GCC 5.4 and 7.5
+- macOS with Apple clang 11.0 using Boost 1.70.0
+- Windows with Visual Studio 2015 using Boost 1.60.0 and Visual Studio 2017
+
+Versions older than the ones listed may not work and are not
+supported; use them at your own risk.
+
+Versions newer than the ones listed above should work; if you
+have problems, please file a bug report via
+[JIRA](https://jira.mongodb.org/browse/CXX/).
+
+## Installation
+
+### Step 1: Install the latest version of the MongoDB C driver.
```

Comment:
I would argue for putting steps 1-3 on their respective OS pages as well, even if the information is duplicated. I think Hugo supports some way of including common files.

My overall opinion is that the main installation documentation should be a "getting started" guide. A user should be able to follow all instructions on one page for one OS using the latest version of the driver. Advanced options should be accessible but separate instead of interspersed within the installation guide.

Reply:
At first I wasn't sure about this, but after implementing it I like the appearance better.